### PR TITLE
Create a variable to receive the color of the percentage relating to your theme

### LINKED ISSUE #4: [THEMES] Create new variable in theme to change color percentage value
Create a variable to receive the color of the percentage relating to your theme

### PR DESCRIPTION
**Pull request by gbcode98**

### CODE REVIEW

**Comment by CriticalNoob02** [comment]
Tem que adicionar essa variavel no type  SvgTheme

File: api/service/svg.py
Comment (CriticalNoob02) [inline api/service/svg.py:1]: Faltou modificar o type da variável, type > svg_service > SvgTheme

**Review by CriticalNoob02 — CHANGES_REQUESTED** [review]

File: api/service/svg.py
Comment (gbcode98) [reply api/service/svg.py:1]: feito.

That's because I was testing him 

image

**Comment by gbcode98** [comment]
@CriticalNoob02 revisar novamente

**Review by CriticalNoob02 — APPROVED** [review]

**Pull request merged**